src/interface/file.rs

```diff
@@ -514,7 +514,9 @@ mod tests {
         let emulated_file = EmulatedFile::new(file_path.clone(), file_content.len()).unwrap();
 
         let mut buffer = vec![0; file_content.len()];
-        let bytes_read = emulated_file.readat(buffer.as_mut_ptr(), buffer.len(), 0).unwrap();
+        let bytes_read = emulated_file
```

Comment:
everything in this file is just rust fmt fixing what I didn't format in the read_at PR